react/game/src/scss/_variables.scss

```diff
@@ -0,0 +1 @@
+$font-family: 'Poppins', sans-serif;
```

Comment:
rename this file and move it to `src/scss/variables/_fontConfig.scss`. Also, move the `_colors.scss` file to variable folder